data/src/main/java/org/openmbee/sdvc/data/domains/global/Branch.java

```diff
@@ -75,4 +76,19 @@ public boolean isInherit() {
     public void setInherit(boolean inherit) {
         this.inherit = inherit;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
there's a generic equals and hashCode in Base, would prefer to try fixing that if it's not adequate instead of adding them to all subclasses

Reply:
Yeah, I'm not sure what side effects these will have since they don't all include all of the fields. These were only meant to get the desired behavior from the recalculateInhertiedPerms() methods.  I can reimplement it a different way.